src/utils/index.js

```diff
@@ -231,4 +231,15 @@ export function getSortedDate(start, end) {
   return isBefore(start, end) ? { start, end } : { start: end, end: start };
 }
 
+export const chunk = (target, size) => {
```

Comment:
Would you mind adding some comments and UT for this function?
Not very clear what is it for.